code/_onclick/click.dm

```diff
@@ -70,6 +70,14 @@
 		var/mob/living/heavy_vehicle/M = loc
 		return M.ClickOn(A, params, src)
 
+	// pAI handling
+	if(istype(loc.loc, /mob/living/bot))
```

Comment:
why loc.loc again?

Reply:
the pAI is inside a bot, inside a card. two locs.